lib/engine/game/g_18_usa/step/track.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/tracker'
+require_relative '../../../step/track'
+require_relative '../../../step/upgrade_track_max_exits'
+
+module Engine
+  module Game
+    module G18USA
+      module Step
+        class Track < Engine::Step::Track
+          include Engine::Step::UpgradeTrackMaxExits
+
+          def check_track_restrictions!(entity, old_tile, new_tile)
+            return if @game.loading || !entity.operator?
+
+            graph = @game.graph_for_entity(entity)
```

Comment:
what's different about this ?

Reply:
This is to support the case of upgrading the iron tiles where there is no town/city to change revenue and no new track to connect to.  I've replaced this with a simpler version that references `super`